entc/integration/edgeschema/ent/schema/tweet.go

```diff
@@ -31,6 +31,6 @@ func (Tweet) Edges() []ent.Edge {
 		edge.From("user", User.Type).
 			Ref("tweets").
 			Through("tweet_user", UserTweet.Type).
-			Comment("The uniqueness is enforced on the edge schema"),
+			Comment("User who authored this tweet. \nThe uniqueness is enforced on the edge schema"),
```

Comment:
No need to change this comment, please. The second sentence is more important for this example.